r4/resource-apis/account/Package.md

```diff
@@ -0,0 +1,79 @@
+
+
+# Account Template
+
+## Template Overview
+
+This template provides a boilerplate code for rapid implementation of FHIR APIs and creating, accessing and manipulating FHIR resources.
+
+
+| Module/Element       | Version |
+|---| --- |
+| FHIR version         | r4 |
+| Implementation Guide | [http://hl7.org/fhir](http://hl7.org/fhir) |
+| Profile URL          |[http://hl7.org/fhir/StructureDefinition/Account](http://hl7.org/fhir/StructureDefinition/Account)|
+
+### Dependency List
+
+| Module | Version |
+| --- | --- |
+|
+
```

Comment:
Should this be empty? If so, can we totally remove this?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates/commit/10d99e6600a555498933a04733525050c16746f1